data/persons.txt

```diff
@@ -616,6 +616,7 @@ ship "Modified Osprey"
 			"hull damage" 3500
 			"heat damage" 24000
 			"ion damage" 24000
+			"weapon jamming damage" 24000
 			"disruption damage" 1200
```

Comment:
"Disruption" is already barely short enough as to not cause issues with attribute name length. "Weapon jamming" as a prefix or suffix for various attribute names (e.g. "damage," "resistance," "protection") will certainly be too long.

Reply:
okay I'll rename it to jamming damage then